x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/text_expansion_callout.tsx

```diff
@@ -182,6 +185,94 @@ export const ModelDeploymentInProgress = ({
 export const ModelDeployed = ({
   dismiss,
   isDismissable,
+  isStartButtonDisabled,
+}: Pick<TextExpansionCallOutState, 'dismiss' | 'isDismissable' | 'isStartButtonDisabled'>) => {
+  const { startTextExpansionModel } = useActions(TextExpansionCalloutLogic);
+
+  return (
+    <EuiPanel color="success">
+      <EuiFlexGroup direction="column" gutterSize="s">
+        <EuiFlexItem grow>
+          <EuiFlexGroup direction="row" gutterSize="s" alignItems="center">
+            <EuiFlexItem grow={false}>
+              <EuiIcon type="checkInCircleFilled" />
+            </EuiFlexItem>
+            <EuiFlexItem grow>
+              <EuiTitle size="xs">
+                <h4>
```

Comment:
Are you sure this should be an `h4`? Is this embedded in a section led by an `h3`? Heading levels are important for accessibility reasons, they are not just for text size/styling: 
https://elastic.github.io/eui/#/guidelines/accessibility#headings-and-landmarks


To check, you can use the [axe devTools chrome plugin](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd)

Reply:
If you change this to an `EuiCallOut` this will fix itself, because callouts take a string in the title prop and automatically apply the correct heading level and styling.

Reply:
@sphilipse  Another concern is the dismissability of the callout with the top right X. Is it something natively supported by `EuiCallOut`?

Reply:
Not natively dismissable, but the title can take a ReactNode so you can essentially add the same content as you already have in the title. But at that point the difference between a callout and a panel is getting a lot smaller, so it might not be worth the effort anymore.

Reply:
And yet another minor detail I noticed is that in `EuiPanel` the edges are rounded but in `EuiCallOut` they are not.

Reply:
Here's my current solution ([commit of changes](https://github.com/elastic/kibana/pull/156080/commits/eb99068fb7cd41a07f6b2069645a9bd796257967)):
- I switched from `EuiPanel` to `EuiCallOut`
  - Because of this switch the edges are no longer rounded like in the design
- I didn't find a way to add a badge and the dismiss X icon into the title of an `EuiCallOut`, so I continued using individual components
- I replaced `EuiTitle` with `EuiText` (with a nested h3) that accepts `color`.
- I replaced `FormattedMessage`-s for title/body texts with `i18n.translate()` inside a `h3/p` tag respectively.

Current screenshots:
![Screenshot 2023-04-28 at 10 44 28 AM](https://user-images.githubusercontent.com/14224983/235188149-5bee0eba-7f4c-4f46-bbc4-61c306484602.png)
![Screenshot 2023-04-28 at 10 46 34 AM](https://user-images.githubusercontent.com/14224983/235188150-5db6c995-788d-46ab-9b66-7fbe06e00d08.png)
![Screenshot 2023-04-28 at 10 59 05 AM](https://user-images.githubusercontent.com/14224983/235188155-23443f32-64bc-4e49-a6ea-0e812959b964.png)
![Screenshot 2023-04-28 at 10 59 14 AM](https://user-images.githubusercontent.com/14224983/235188156-17a7f3f0-363f-40a2-a518-7c6b5fe6b912.png)

cc @sphilipse @julianrosado

Reply:
I'm good with all the changes, thank you for running it down!

Small nitpick

![CleanShot 2023-04-28 at 20 15 24](https://user-images.githubusercontent.com/4490333/235272385-9e024cea-b748-44f4-ad45-3c4743635356.png)

1. Looks like the button are flex-ed to the max and is taking up way too much width. I believe the solution is to wrap it in a div? See: https://eui.elastic.co/#/layout/flex#flex-items-are-also-flex (or maybe this [one](https://eui.elastic.co/#/layout/flex#turn-off-item-stretching))

2. Gutter size for between the buttons and link should be `gutterSize="m"` on that flex group

After that you are good to go!

Reply:
@julianrosado Updated the button and the gutter:
![Screenshot 2023-04-30 at 6 31 41 PM](https://user-images.githubusercontent.com/14224983/235378982-c545f02b-590a-40ae-9e59-b2e79da26d37.png)